src/Microsoft.Diagnostics.DebugServices.Implementation/Runtime.cs

```diff
@@ -111,21 +112,23 @@ public string GetCDacFilePath()
             return _cdacFilePath;
         }
 
+
         public string GetDacFilePath(out bool verifySignature)
         {
-            verifySignature = false;
             if (_settingsService.ForceUseContractReader)
             {
+                verifySignature = false;        // Don't verify signature when using the CDAC
```

Comment:
Modify the cache too.

Reply:
I commented why it shouldn't be cached for the cdac.